lib/erb_lint/cache.rb

```diff
@@ -76,7 +80,7 @@ def checksum(filename, file_content)
       mode = File.stat(filename).mode
 
       digester.update(
-        "#{mode}#{config.to_hash}#{ERBLint::VERSION}#{file_content}",
+        "#{mode}#{config.to_hash}#{ERBLint::VERSION}#{@runner_checksum}#{file_content}",
```

Comment:
Note that this will bust all caches on release since `Runner#checksum` will return `Digest::SHA1.new.hexdigest` even if there are no linters with checksum methods. That method can be refactored to return `nil` in the default / empty case if desired.